data/docs.yml

```diff
@@ -80,8 +80,8 @@
       user: apache
       repo: skywalking-client-js
       docs:
-        - version: v0.4.0
-          link: https://github.com/apache/skywalking-client-js/tree/v0.4.0
+        - version: v0.5.0
+          link: https://github.com/apache/skywalking-client-js/tree/v0.5.0
```

Comment:
Take a reference from backend nodejs agent, add npm link please.

Reply:
![image](https://user-images.githubusercontent.com/26432832/118344254-ded5b480-b55f-11eb-9532-0303b6eda737.png)
It's been added here.